ts/types/Contact.ts

```diff
@@ -101,5 +101,11 @@ export function contactSelector(
 
 export function getName(contact: Contact): string | null {
   const { name, organization } = contact;
-  return (name && name.displayName) || organization || null;
+  const displayName = (name && name.displayName) || null;
```

Comment:
Is `name && …` required since `Contact::name` is marked as required? Is it optional? If so, can we update the type definition above?

Reply:
Yeah, it's probably time to get rid of all required elements of that type definition.

Reply:
You can always wrap it in `Partial<…>` for all protobuf related interfaces or better yet, we should check if we can reuse the ones that are generated for us.

Reply:
I just went and looked at it - we still have some required things. `value` and `type` are always available on `Phone` and `Email`, and `PostalAddress` always has `type`. So I left those.